eng/packages/http-client-csharp/package.json

```diff
@@ -9,11 +9,11 @@
   "scripts": {
     "clean": "rimraf ./dist ./emitter/temp && dotnet clean ./generator",
     "build:emitter": "tsc -p ./emitter/tsconfig.build.json",
-    "build:generator": "dotnet build ./generator",
-    "build": "npm run build:emitter && npm run build:generator && copyfiles -u 5 ./node_modules/@typespec/http-client-csharp/dist/generator/Microsoft.Generator.CSharp.runtimeconfig.json ./dist/generator",
+    "build:generator": "dotnet build ./generator && copyfiles -u 5 ./node_modules/@typespec/http-client-csharp/dist/generator/Microsoft.Generator.CSharp.runtimeconfig.json ./dist/generator",
+    "build": "npm run build:emitter && npm run build:generator",
     "test:emitter": "vitest run -c ./emitter/vitest.config.ts",
-    "test:generator": "dotnet test ./generator",
-    "test": "npm run test:emitter && npm run test:generator",
+    "test:generator": "dotnet test ./generator --logger \"trx;LogFileName=debug.trx\"",
+    "test": "npm run test:generator && npm run test:emitter",
```

Comment:
is there a reason to change the order here?

Reply:
Emitter tests were failing and generator tests weren't running.  We will want to put the faster test suite first.